lib/addons/topics-api.ts

```diff
@@ -0,0 +1,40 @@
+import OptableSDK from "../sdk";
+
+declare module "../sdk" {
+    export interface OptableSDK {
+        tryTopicsAPI: () => void;
+    }
+}
+
+declare global {
+    interface Document {
+        browsingTopics: () => Promise<Array<{
+            configVersion: string;
+            modelVersion: string;
+            taxonomyVersion: string;
+            topic: number;
+            version: string;
+        }>>;
+        featurePolicy:{ allowsFeature: (feature:string) => boolean };
+    }
+}
+
+OptableSDK.prototype.tryTopicsAPI = async function () {
+    if (!sessionStorage.topics_fetched && 'browsingTopics' in document && document.featurePolicy.allowsFeature('browsing-topics')) {
+        const topicsArray = await document.browsingTopics();
+        if (topicsArray.length > 0) {
+            const topics: string[] = [];
+            for (let topic of topicsArray) {
+                topics.push('taxonomy version: ' + topic.taxonomyVersion + ', topic id: ' + topic.topic);
```

Comment:
I'd recommend a format that is easier to parse on the backend, maybe `{version}:{id}` ?

Reply:
this won't mean much for the end user
['2:57', '2:192']